homeassistant/components/deconz/alarm_control_panel.py

```diff
@@ -100,16 +121,22 @@ class DeconzAlarmControlPanel(DeconzDevice, AlarmControlPanelEntity):
     """Representation of a deCONZ alarm control panel."""
 
     TYPE = DOMAIN
+    _device: AncillaryControl
 
     _attr_code_format = FORMAT_NUMBER
     _attr_supported_features = (
         SUPPORT_ALARM_ARM_AWAY | SUPPORT_ALARM_ARM_HOME | SUPPORT_ALARM_ARM_NIGHT
     )
 
-    def __init__(self, device, gateway) -> None:
+    def __init__(
+        self,
+        device: AncillaryControl,
+        gateway: DeconzGateway,
+        alarm_system: AlarmSystem,
+    ) -> None:
```

Comment:
Change of api (more than just typing). Should be in the PR description?

Reply:
This change is to not need to do additional typing on the alarm_system attribute, which would've been necessary in every call of self.alarm_system

Reply:
Ok, I can only see [one reference to alarm_system](https://github.com/home-assistant/core/search?q=alarm_system), but I see how this makes it better and it doesn't affect any other parts of HA.

Reply:
All the calls to arm/disarm uses the alarm system